### DISCUSSION TITLE
I love the design and this needs more attention

### DISCUSSION THREAD

**Discussion by Wamy-Dev** [riazer-lab/sira#61] [General]
I honestly really love the design language of this, and even though you dont have many components. I am thinking of using it in an upcoming project for sure. Really good work on this!

**Comment by eyeix** [comment]
@Wamy-Dev  Thanks for your feedback! 💪
I just created this one not long ago, I will create more components & snippets into it in the future.
Wish more people would love this, you can recommand this for more developers.
